src/ripple/protocol/Feature.h

```diff
@@ -348,6 +348,7 @@ extern uint256 const fixNonFungibleTokensV1_2;
 extern uint256 const fixNFTokenRemint;
 extern uint256 const fixReducedOffersV1;
 extern uint256 const featureClawback;
+extern uint256 const fixFillOrKillOnFlowCross;
```

Comment:
I probably wouldn't mention "FlowCross". Just `fixOfferFillOrKill`. There's only one offer crossing implementation, there's no need to mention it by name.